backend/v3/storage/database/dbmock/database.mock.go

```diff
@@ -21,6 +21,7 @@ import (
 type MockPool struct {
 	ctrl     *gomock.Controller
 	recorder *MockPoolMockRecorder
+	isgomock struct{}
 }
```

Comment:
Why was the generated struct for the Pool type changed? According to the new gen_mock.go, I'd expect only changes related to the `Client` and `Connection` types. @adlerhurst could you please check if `mockgen --version` prints v0.4.0?